api-catalog-ui/frontend/src/components/ServicesNavigationBar/ServicesNavigationBar.jsx

```diff
@@ -55,18 +70,15 @@ export default class ServicesNavigationBar extends Component {
         const { match, services, searchCriteria } = this.props;
         const hasTiles = services && services.length > 0;
         const hasSearchCriteria = searchCriteria !== undefined && searchCriteria !== null && searchCriteria.length > 0;
-        let selectedTab = Number(localStorage.getItem('selectedTab'));
+        const url = window.location.href;
+        const parts = url.split('/');
+        const serviceId = parts[parts.length - 1];
+        let selectedTab;
```

Comment:
can this be assigned to 0 and then used without OR on line 121? value={selectedTab || 0}

Reply:
done